students/shayna_andersonhill/lesson03/mailroom.py

```diff
@@ -0,0 +1,82 @@
+#!/Library/Frameworks/Python.framework/Versions/3.6/bin/python3
+#Shayna Anderson-Hill
+#Mailroom Part 1
+#03-04-2018
+
+donations = [['Shayna Hill', [50, 50, 50]], ['Brandon Aleson', [200]],
+        ['Lisa Rodriguez', [10, 20]], ['Marge Simpson', [100]],
+        ['Charlie Smith', [40]]]
+
+def print_names():
+    print(name_list)
+
+def add_name(name):
+    name_list.append(name)
+    donations.append([name, []])
+
+def add_new_name_donation(name, donation):
+    if donations[-1][0] == name:
+        donations[-1][1].append(donation)
+
+def add_old_name_donation(name, donation):
+    donations[counts][1].append(donation)
+
+def formatter():
+    heading = ['Donor Name', '| Total Given', '| Num Gifts', '| Average Gift']
+    print('{:20}{:>15}{:>15}{:>15}'.format(*heading))
+    print('------------------------------------------------------------------')
+    for donor, total, num, average in zip(donors, total_given, num_gifts,
+        average_gifts):
+        print('{:20}    ${:12.2f}{:>12}   ${:>12.2f}'.format(donor, total, num, average))
+
+
+
+
+
+if __name__ == '__main__':
+    response = input('Would you like to "Send a Thank You", "Create a Report", or "quit"?')
+    name_list = [i[0] for i in donations]
+
+    donors = []
+    for i,j in donations:
```

Comment:
`i` and `j` don't tell me anything about what you are iterating threw,  use something like `for name, donation in donations`